index.js

```diff
@@ -0,0 +1,2 @@
+const nombreUsuario = prompt("¿Cuál es tu nombre?")
```

Comment:
hay que optimizar la pregunta del prompt

```suggestion
const nombreUsuario = prompt("Ingrese su nombre.")
```